Report.java

```diff
@@ -202,4 +205,16 @@ private void printSpriteCounts(PrintWriter printW, int i)
             printW.println();
         }
     }   
+
+    /**
+     * Method to get the current datetime for report.
+     *
+     * @return formatted datetime
+     */
+    public String getReportDateTime()
+    {
+        DateFormat dateFormat = new SimpleDateFormat("yyyyMMddHHmmss");
```

Comment:
Can we change the format of the date to make it more readable? When I look at the file name it's: Report-2017425180303. I had to look at the code to figure out how it was formatted since it just looked like random numbers at first glance mainly because I wasn't thinking about it being military time. Maybe some dashes and dots would make it look more readable.